projects/Objects/HomogenousTree/src/com/valentin/HTree.java

```diff
@@ -0,0 +1,144 @@
+package com.valentin;
+
+
+import java.util.Vector;
+
+public class HTree {
+    Vector<Node> nodesVector=new Vector();
```

Comment:
Note that the idea here is tree to be constructed manually without using of java.util.* package or something related. 

The node class should be something like this: 
```java
public class Node {
   private Node leftNode;
   private Node rightNode;
   private int key;
   private Object value;
}

```